.github/workflows/publish-docs.yml

```
@@ -90,14 +84,13 @@ jobs:
               echo "ENVIRONMENT=preview" >> $GITHUB_ENV
               echo "DEPLOY_PROD=false" >> $GITHUB_ENV
           fi
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=${{ env.ENVIRONMENT }} --token=${{ secrets.VERCEL_TOKEN }}
+      - name: Link Vercel Project
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)